bin/yunoprompt

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# Fetch ips
+ip=$(hostname --all-ip-address)
+
+# Fetch SSH fingerprints
+i=0
+for key in /etc/ssh/ssh_host_*_key.pub ; do 
+    output=$(ssh-keygen -l -f $key)
+    fingerprint[$i]=" - $(echo $output | cut -d' ' -f2) $(echo $output| cut -d' ' -f4)"
+    i=$(($i + 1))
+done
+
+#
+# Build the logo
+#
+
+LOGO=$(cat << 'EOF'
+ '.                        '                                    ''     -d.    
+ /M+                       h-   .shh/           //            /NMy-    hMdosso
+ 'MN'   /' '.    -'  :+    N:  .Nmyym     yo   .MN'  omNds:   :mN' .sydMMMNds+
+  sMh:/dN: :M'   m:  oMh' .M:  dy   h'    MM:  'Mo  oMh:-sMh    /ddNdyyNM'    
+   .sMMy'  /M'  /M-  sMMd/sM- -Ms  +M+    MM:  +M/  mM'  -Md     'NM.  hM.    
+     mM    .M- :NN   yMMMMMM: .dMNNMd' -/oMMmhmMMh  /msosNM/   ::oMM.  +M:    
+    'MN     sMNMM+   mN:.+mM+   -+o/   :hMMm+- 'oN-   :oyo-    'yho.    -     
+     hy      /yy:    :-    -.            -Nh      '                           
+      .                                   
+EOF
+)
+
+# ' Put a quote in comment to make vim happy about syntax highlighting :s
+
+#
+# Build the actual message
+#
+
+LOGO_AND_FINGERPRINTS=$(cat << EOF
+"$LOGO"
+
+IP: ${ip}
+SSH fingerprints:
+${fingerprint[0]}
+${fingerprint[1]}
+${fingerprint[2]}
+${fingerprint[3]}
+${fingerprint[4]}
+EOF
+)
+
+if [[ ! -f /etc/yunohost/installed ]]
+then
+    if [[ ! -f /etc/yunohost/from_script ]]
```

Comment:
This from_script condition,  display the boot prompt only for iso images / arm images (if they removed the from_script as they should do). 
I don't know if it's a bad idea to display that on VPS ? I think I agree with you, it could be on all settings. Don't forget I had adapted an already existing works...